content/applications/sales/sales/products_prices/loyalty_discount.rst

```diff
@@ -0,0 +1,188 @@
+loyalty programs that customers can use for online and in-store shopping. These programs offer more
+varied, public, and time-sensitive pricing options than :doc:`pricelists <prices/pricing>`.
+
+Configure the settings
+======================
+
+To begin using discount and loyalty programs, navigate to :menuselection:`Sales --> Configuration
+--> Settings`. Under the :guilabel:`Pricing` heading, activate the :guilabel:`Discounts, Loyalty &
+Gift Card` setting by checking the box next to the feature. Finally, click :guilabel:`Save` to save
+the changes.
+
+Configure discount and loyalty programs
+=======================================
+
+To create discount and loyalty programs, go to :menuselection:`Sales --> Products --> Discount &
+Loyalty`.
+
+If no discount or loyalty programs have been created yet, Odoo provides a choice of templates to
+help create the first program. Choose one of the template cards, or click :guilabel:`New` to create
+a new program from scratch.
+
+Or, if there are already existing programs, select an existing program to edit it.
+
+.. image:: loyalty_discount/price-discount-loyalty.png
+   :align: center
+   :alt: Discount and loyalty program template cards.
+
+.. note::
+   Templates only appear when no programs have been created, and they disappear once the first
+   program is created.
+
+Creating or editing a program opens the program form.
+
+The program form contains the following options:
+
+- :guilabel:`Program Name`: Enter the name of the program. The program name is *not* visible to the
+  customer.
+- :guilabel:`Program Type`: Select the desired :ref:`program type
+  <sales/pricing_management/program-types>`.
+- :guilabel:`Currency`: Select the currency used for the program.
+- :guilabel:`Points Unit`: Enter the name of the points used for the :guilabel:`Loyalty Cards`
+  program (e.g. `Loyalty Points`). The points unit name *is* visible to the customer. This field is
+  only available when the :guilabel:`Program Type` is set to :guilabel:`Loyalty Cards`.
+- :guilabel:`Validity`: Select the date until which the program is valid. Leave this field blank for
+  no end date, meaning the program is always valid and does not expire.
+- :guilabel:`Limit Usage`: Check this box and enter a number to limit the number of times the
+  program can be used during the :guilabel:`Validity` period.
+- :guilabel:`Company`: In the case of multiple companies, choose the company for which the program
+  is available.
+- :guilabel:`Available On`: Select the app(s) on which the program is available.
+- :guilabel:`Website`: Select the website(s) on which the program is available. Leave this field
+  blank to make it available on all websites.
+- :guilabel:`Point of Sale`: Select the point(s) of sale at which the program is available. Leave
+  this field blank to make it available at all :abbr:`PoS (Point of Sale)`.
+
+.. image:: loyalty_discount/price-programs.png
+   :align: center
+   :alt: Program options
```

Comment:
Please add more contextual information to the alt tags throughout the doc as well as a period
```suggestion
   :alt: Program options on the program form in Odoo.
```